pkg/build/builder/docker.go

```diff
@@ -360,10 +360,10 @@ func getDockerfilePath(dir string, build *buildapi.Build) string {
 }
 func parseDockerfile(dockerfilePath string) (*parser.Node, error) {
 	f, err := os.Open(dockerfilePath)
-	defer f.Close()
```

Comment:
Both are equivalent, although the previous version was more readable. Being able to have the deferred close call next to open is the reason for this construct in golang.